### PR TITLE
Add service account flag to namespaced resources

### PR DESCRIPTION
**Pull request by ncarlson**

### CODE REVIEW

File: docs/kp_secret_create.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 ## kp secret create
 
-Create a secret configuration
+Create a secret for a service account
```
Comment (matthewmcnew) [inline docs/kp_secret_create.md:3]:
```suggestion
Create a secret and link it to a service account
```
?

File: docs/kp_secret_list.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
 ## kp secret list
 
-List secrets
+List secrets for a service account
```
Comment (matthewmcnew) [inline docs/kp_secret_list.md:3]:
```suggestion
List secrets attached to a service account?
```

**Comment by matthewmcnew** [comment]
Looks good! Curious about some of the help text. Love to get other opinions.

File: docs/kp_secret_list.md (hunk 2 above)
Comment (ncarlson) [reply docs/kp_secret_list.md:3]: How about `List secrets linked to a service account`?

File: docs/kp_secret_create.md (hunk 1 above)
Comment (ncarlson) [reply docs/kp_secret_create.md]: How about `Create a secret and add it to a service account`?

File: docs/kp_secret_list.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 
 ### Synopsis
 
-Prints a table of the most important information about secrets in the provided namespace.
+List secrets for a service account in the provided namespace.
```
Comment (matthewmcnew) [inline docs/kp_secret_list.md:7]: I like this.

File: docs/kp_secret_list.md (hunk 2 above)
Comment (ncarlson) [reply docs/kp_secret_list.md:3]: Attached. I'm good with attached.

File: docs/kp_secret_create.md (hunk 1 above)
Comment (matthewmcnew) [reply docs/kp_secret_create.md]: I like that.

**Review by matthewmcnew — APPROVED** [review]

**Comment by tylerphelan** [comment]
let's merge!

**Pull request merged**